cmd/unused-exporter/exporter.go

```diff
@@ -286,10 +291,14 @@ func lastUsedTS(d unused.Disk) float64 {
 }
 
 func getRegionFromZone(p unused.Provider, z string) string {
-	if strings.ToLower(p.Name()) == "azure" {
-		return z
+	var region string
+	switch p.Name() {
+	case azure.ProviderName:
+		region = z
+	case gcp.ProviderName:
+		region = z[:strings.LastIndex(z, "-")]
+	case aws.ProviderName:
+		region = z[:len(z)-1]
 	}
-
-	// Drop the last character to get the region from the zone for GCP and AWS
-	return z[:len(z)-1]
+	return region
```

Comment:
just curious, is there a reason to not use a default case and choose to jump out of the switch instead?

Reply:
I suppose because it's semantically the same.

Reply:
Nice!